tests/pds/dataservice_test.go

```diff
@@ -1065,3 +1075,112 @@ func DeployInANamespaceAndVerify(nname string, namespaceID string) []string {
 
 	return cleanup
 }
+
+var _ = Describe("{RollingRebootNodes}", func() {
+	JustBeforeEach(func() {
+		StartTorpedoTest("PDS: RollingRebootNodes", "Reboot node(s) while the data services will be running", nil, 0)
+	})
+
+	It("has to deploy data service and reboot node(s) while the data services will be running.", func() {
```

Comment:
are we not loading the data service while rebooting the nodes ?
we should have a Step for generating load on the data service before rebooting nodes.
Refer: RestartPXPods testcase

Reply:
Add a TODO comment for tracking purpose and also create a task ticket for the same.

Reply:
Ref ticket: https://portworx.atlassian.net/browse/PA-403